module/id/fixed_provider_test.go

```diff
@@ -23,11 +24,31 @@ func TestFixedIdentifierProvider(t *testing.T) {
 	in := identifiers[rand.Intn(10)]
 	out := unittest.IdentifierFixture()
 
-	assert.True(t, contains(fp.Identifiers(), in))
-	assert.False(t, contains(fp.Identifiers(), out))
+	require.True(t, contains(fp.Identifiers(), in))
```

Comment:
why change `assert` to `require`?

Reply:
I'm fine with either way. 

my understanding is `assert` will mark the test as failing and continue so you can see the full list of issues for the test. `require` will fail immediately, which is useful for cases where a condition breaks the rest of the test (e.g. `require.NoError()` when creating an object needed for test).